library/src/com/google/maps/android/data/geojson/GeoJsonLayer.java

```diff
@@ -0,0 +1,164 @@
+package com.google.maps.android.data.geojson;
+
+import com.google.android.gms.maps.GoogleMap;
+import com.google.android.gms.maps.model.LatLngBounds;
+import com.google.maps.android.data.Layer;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import android.content.Context;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.HashMap;
+
+/**
+ * A class that allows the developer to import GeoJSON data, style it and interact with the
+ * imported data.
+ *
+ * To create a new GeoJsonLayer from a resource stored locally
+ * {@code GeoJsonLayer layer = new GeoJsonLayer(getMap(), R.raw.resource,
+ * getApplicationContext());}
+ *
+ * To render the imported GeoJSON data onto the layer
+ * {@code layer.addLayerToMap();}
+ *
+ * To remove the rendered data from the layer
+ * {@code layer.removeLayerFromMap();}
+ */
+public class GeoJsonLayer extends Layer {
+
+    private final GeoJsonRenderer mRenderer;
+
+    private LatLngBounds mBoundingBox;
+
+    private GoogleMap mMap;
```

Comment:
Looks like `mMap` isn't used and can be removed.  Or, it should be assigned in the contructors.